hermes-guide-days-website/src/app/components/header/header.component.scss

```diff
@@ -0,0 +1,141 @@
+@import url(https://fonts.googleapis.com/css?family=Montserrat);
+
+.topnav {
+    overflow: hidden;
+    background-color: #cc3c3b;
```

Comment:
please merge your branch with Narcis' changes and add the colors to the variables file and use them